packages/babel-helpers/src/index.js

```diff
@@ -238,7 +243,13 @@ function loadHelper(name) {
     }
 
     const fn = () => {
-      return t.file(helper.ast());
+      const ast = t.file(helper.ast());
+      return new File(
+        {
+          filename: `babel-helper://${name}`,
```

Comment:
`babel-helper://` is used to denote that this file is a babel helper snippet thus does not physically exist.